src/traverse.js

```diff
@@ -17,7 +17,7 @@ export function traverse(schema, options, doc, context) {
   options.ticks -= 1;
   
   if (options.ticks === 0) {
-    throw Error('Schema size exceeded');
+    throw Error(`Schema size exceeded: over ${options.startingTicks} elements`);
```

Comment:
Is there a more descriptive phase/word than elements we could use? 

To be fair I can not think of any.

Reply:
Properties is probably better. Working on it.